openedx/core/djangoapps/schedules/migrations/0010_remove_null_blank_from_schedules_date.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2019-12-18 16:18
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('schedules', '0009_schedule_copy_column_values'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
I believe this line should also have tipped us off that an alter statement we would not want to run would be executed.